includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -536,7 +546,7 @@ private function process_webhook_dispute_created( $event_body ) {
 			);
 		}
 
-		$this->order_service->mark_payment_dispute_created( $order, $dispute_id, $reason );
+		$this->order_service->mark_payment_dispute_created( $order, $dispute_id, $amount, $reason, $due_by );
```

Comment:
Idea: since we are not shipping human-readable dispute reasons in this PR, what do you think about wrapping the `$reason` key in quotes to make it more readable in the meantime.

E.g. `Payment has been disputed for 110.00 € with reason "fraudulent".`

Reply:
Example:

![Uploading Screenshot 2023-06-07 at 12.31.18 PM.png…]()